go.mod

```diff
@@ -3,10 +3,13 @@ module go.uber.org/fx
 go 1.13
 
 require (
+	github.com/benbjohnson/clock v1.3.0
 	github.com/stretchr/testify v1.7.0
 	go.uber.org/dig v1.12.0
 	go.uber.org/goleak v1.1.11
 	go.uber.org/multierr v1.5.0
 	go.uber.org/zap v1.16.0
 	golang.org/x/sys v0.0.0-20210903071746-97244b99971b
 )
+
+replace github.com/benbjohnson/clock => github.com/abhinav/clock v1.3.1-0.20211118204451-38de6f8a55b9
```

Comment:
We probably will not land this `replace` directive.
I'd like to wait a few days for the change to be merged upstream.